lib/class-sp-sync-manager.php

```diff
@@ -166,6 +166,26 @@ public function get_posts( $args = array() ) {
 	 * @return bool
 	 */
 	public function do_index_loop() {
+
+		/**
```

Comment:
Nitpicky, but IIRC a multiline comment should only start with two asterisks if it is explicitly a docblock.

Reply:
:til: